Attendance.csv

```diff
@@ -23,3 +23,8 @@ Date,Session,Attendance,Notes,
 2017-06-14,Coworking: Unit Testing,4,1 neuroscience 2 physics 1 math/statistics,
 2017-06-21,Continuous Integration & Github,7,1 economics 1 math/statistics 1 physics 1 neuroscience 1 physiology 1 nutritional science,
 2017-06-28,LD: Data Visualization,9,2 physics 1 physiology 1 banker 1 neuroscience 1 eeb 1 csb 1 biophysics 1 computer science,
+2017-07-05,Intermediate SQL,,,
+2017-07-12,Coworking,1,1 physiology,
+2017-07-19,dplyr and magrittr,6,1 banker 2 nutritional sciences 1 physics ... ,
```

Comment:
@linamnt do you remember who attended this session (2017-07-19)? I have four people in the notes section... but I don't remember the other ones.